internal/pkg/local/save.go

```diff
@@ -25,7 +25,7 @@ func (m *Manager) saveObject(record model.Record, object model.Object, changedFi
 
 	w := modelWriter{
 		Manager:         m,
-		LocalSaveRecipe: &model.LocalSaveRecipe{ChangedFields: changedFields, Object: object, Record: record},
+		LocalSaveRecipe: &model.LocalSaveRecipe{ChangedFields: changedFields, Object: object.Clone(), Record: record},
```

Comment:
Skusal som stiahnut `shared config` so starou verziou, a potom pravit `pull --force` s novou verziou, teda to premigrovat.
Pull presiel v pohode, ale ked som spustil nasledne `diff`, tak ukazoval rozdiely v shared codes.

Skumal som to, a zistil som, ze su problemy s klonovanim.
Vacsinou v testoch generujeme tento `LocalSaveRecipe` manualne, tak sa to neprejavilo.
Neprejavi sa to ani pri jednoduchych operaciach, kde sa hned skonci program, ... takze nevalidny interny stav nevidno.

Prejavi sa to pri `pull`, po ktorom sa robi `rename`:
- pri `pull` sa ulozi vsetko spravne, ... ale chybne sa upravi interny stav, ... v transformation `Blocks` uz nie su IDcka ale cesty ku shared codes
- pri rename shared codes sa maju transformacie znova ulozit, ... co sa stane, ... ale kedze tam uz nie su IDcka, tak sa ulozia opat stare cesty.